docs/guides/operator-guide-azure-enclave.md

```diff
@@ -9,129 +9,312 @@ sidebar_position: 18
 
-Example of running the image:
+Follow these steps:
 
-```
-docker run \
-        --device /dev/sgx/enclave --device /dev/sgx/provision \
-        -p 8080:8080 \
-        -p 9091:9091 \
-        ghcr.io/iabtechlab/uid2-operator-azure-occlum:dev
-```
+1. Update the `vault.parameters.json` file with the following required values:
 
-#### Direct Invocation of Occlum (Advanced)
+   | Parameter | Description |
+   | :--- | :--- |
+   | `vaultName` | The name of the key vault for hosting the operator key secret. The name you choose must be globally unique. |
+   | `operatorKeyValue` | The `OPERATOR_KEY` secret value, which you received from the UID team as part of account setup (see [Set Up UID2 Operator Account](#set-up-uid2-operator-account)). This value is unique to you, and acts as a password: keep it secure and secret. |
 
-You can also run the occlum enclave either from host VM or from your own docker image:
+2. (Optional) If you don't want to accept the defaults, update the `vault.parameters.json` file with the following values. These parameters have default values and in most cases you won't need to make any updates.
 
-```
-tar xvf uid2-operator.tar.gz
-cd uid2-operator
-occlum run /bin/launcher
-```
+    Parameter | Description |
+   | :--- | :--- |
+   | `operatorIdentifier` | The name of the managed identity that will launch the container.<br/>Default: `uid-operator`. |
+   | `operatorKeyName` | The operator key secret name.<br/>Default: `operator-key`. |
+
+3. Run the following command to trigger the deployment:
+
+   ```
+   az deployment group create --name vault --resource-group {RESOURCE_GROUP_NAME} --parameters vault.parameters.json  --template-file vault.json
+   ```
+
+### Set Up the VPC Network
+
+The next step is to set up the VPC network.
+
+The following diagram illustrates the virtual private cloud that hosts a UID2 Private Operator in Microsoft Azure.
+
+![VPC Network](images/operator-azure-drawio.png)
+
+Follow these steps:
+
+1. (Optional) If you don't want to accept the defaults, update the `vnet.parameters.json` file with the following values. These parameters have default values and in most cases you won't need to make any updates.
+
+    Parameter | Description |
+   | :--- | :--- |
+   | `vnetName` | The virtual network name.<br/>Default: `unified-id-network` |
+   | `computeSubnetName` | The name of the subnet that runs the UID2 Operator.<br/>Default: `unified-id-subnet-operators` |
+   | `gatewaySubnetName` | The name of the subnet that runs the UID2 Gateway.<br/>Default: `unified-id-subnet-gateway` |
+   | `VnetAddressPrefix` | The vnet address prefix.<br/>Default: `10.0.0.0/20` |
+   | `computeSubnetPrefix` | The vnet address prefix of the subnet that is delegated to run the UID2 Operator.<br/>Default: `10.0.0.0/24` |
+   | `gatewaySubnetPrefix` | The vnet address prefix of the subnet that runs the UID2 Gateway.<br/>Default: `10.0.0.0/28` |
```

Comment:
Change `gatewaySubnetPrefix` default to `10.0.1.0/28`, there's a mistake in original doc.

Reply:
Done.